tests/foreman/ui/test_audit.py

```diff
@@ -171,8 +171,10 @@ def test_positive_delete_event(session, module_org):
     architecture = entities.Architecture().create()
     architecture.delete()
     with session:
-        values = session.audit.search('type=architecture and action=delete')
-        assert values['action_type'] == 'deleted'
+        values = session.audit.search(
+            'type=architecture and action=destroy'
+        )
```

Comment:
black insists on having that in one line, causing travis to fail

Reply:
ah yes, sure, will fix this right away